### ISSUE TITLE
[Game SDK] Can't Clear Activity

### ISSUE THREAD

**Issue by LennyPhoenix** [discord/gamesdk-and-dispatch#114]
**Describe the bug**
When attempting to clear the activity in version 3.2.0, it returns an `InvalidPayload` error, with this being thrown in the logging hook: `ResponseError { code: InvalidPayload, message: "child \"activity\" fails because [child \"supported_platforms\" fails because [\"supported_platforms\" must contain at least 1 items]]" }`.

**Steps to reproduce**
Update the activity to anything, then run `clear_activity`, and it will throw this error.

**Expected behavior**
The activity to be cleared.

**Implementation specifics**
- Language: C
- Game Engine: N/A

**Comment by Jump-Suit** [comment]
3.2.0 isn't a documented version. So you should be aware of the risk by using something undocumented. 3.1.0 is the last "documented" one, see Mason's announcement here: https://discord.com/channels/613425648685547541/697216494760886393/755850401848229988

**Comment by LennyPhoenix** [comment]
Yeah I'm aware that 3.2.0 is unstable, just made this issue so that its recorded somewhere :P
